src/pages/Home.jsx

```diff
@@ -0,0 +1,96 @@
+import "../CSS/homepage.css";
+import { IMAGES } from "../images/index";
+import { Link } from "react-router-dom";
+const Home = () => {
+	return (
+		<div className="main-container main-container-gutter">
+			<div className="catergory-container gap-l grid-6-column">
+				<div className="img-container card-shadow-dark">
+					<img
+						src={IMAGES.duneBookCover}
+						className="img-responsive"
+						alt="product image"
+					/>
+					<div className="text-overlay">Accessories</div>
+				</div>
```

Comment:
This part of the code is repeated 6 times for all 6 categories. Could we rather have a Category Card Component to not repeat the code?

Reply:
Thanks for pointing it out I copy-pasted and forgot to refactor!